src/main/java/seedu/address/model/person/Memo.java

```diff
@@ -35,6 +35,9 @@ public Memo(String memo) {
 
     /**
      * Returns true if a given string is a valid memo.
+     *
+     * @param memo memo.
```

Comment:
Can be more descriptive in writing what's a memo. Maybe somewhere along the line of "memo string that is checked for validity".